src/htm/algorithms/SpatialPooler.cpp

```diff
@@ -463,17 +463,22 @@ void SpatialPooler::compute(const SDR &input, const bool learn, SDR &active) {
   input.reshape(  inputDimensions_ );
   active.reshape( columnDimensions_ );
   updateBookeepingVars_(learn);
+
+  tOverlap.start();
```

Comment:
Nitpick: don't leave these debugging / benchmarking statements all over the place.